HTTP2-Tests.package/H2StreamTests.class/instance/testIncomingTraffic_HCPCDD.st

```diff
@@ -0,0 +1,19 @@
+testing
+testIncomingTraffic_HCPCDD
```

Comment:
Abkürzungen sind in Smalltalk/Squeak nicht üblich, vor allem nicht, wenn sie absolut nicht offensichtlich sind.

Reply:
Schlag mir einen besseren Weg vor unterschiedliche Abfolgen von hereinkommenden Frames zu benennen und ich werde sie mit Freude übernehmen. Dieses Schema ist einfach das kompakteste Schema was den Inhalt vollständig abbildet, welches mir einfiel.